README.md

```diff
@@ -57,114 +59,97 @@ jobs:
-- name: Run dotTEST analyzer
+# Runs Parasoft dotTEST and generates the reports.
+- name: Run dotTEST
   id: dottest
   uses: parasoft/run-dottest-analyzer@1.0.0
   # ...
 
-# Upload analysis results to GitHub.
-- name: Upload results to GitHub
+# Uploads analysis results in the SARIF format, so that they are displayed as GitHub code scanning alerts.
+- name: Upload results (SARIF)
   uses: github/codeql-action/upload-sarif@v1
   with:
     sarif_file: ${{ steps.dottest.outputs.report }}
 
-# Archive reports from analysis as job artifacts.
-- name: Upload report artifacts
+# Uploads an archive that includes all report files (.xml, .html, .sarif).
+- name: Archive reports
   uses: actions/upload-artifact@v2
   with:
     name: Report files
     path: ${{ steps.dottest.outputs.reportDir }}/*.*
 ```
 
 ## Configuring Analysis with dotTEST
-You can configure analysis with Parasoft dotTEST to meet your organization's policies in one of the following ways:
- - By configuring options directly in Parasoft dotTEST tool. See [Parasoft dotTEST User Guide](https://docs.parasoft.com/display/DOTTEST20202) for details.
- - By customizing the dotTEST action directly in your workflow on GitHub with additional parameters. See [Optional Parameters](#optional-parameters) for a complete list of available parameters. 
+You can configure analysis with Parasoft dotTEST in one of the following ways:
+ - By customizing the `Run dotTEST` action directly in your GitHub workflow. See [Action Parameters](#action-parameters) for a complete list of available parameters.
+ - By configuring options in Parasoft dotTEST tool. We recommend creating a `dottestcli.properties` file that includes all the configuration options and adding the file to dotTEST's working directory  - typically, the root directory of your repository. This allows dotTEST to automatically read all the configuration options from that file. See [Parasoft dotTEST User Guide](https://docs.parasoft.com/display/DOTTEST20202) for details.
  
 ### Examples
 This section includes practical examples of how the dotTEST action can be customized directly in the YAML file of your workflow. 
 
-Note that by default, the `workingDir` parameter points to workspace root, which allows you to use relative paths while specifying them in various [parameters](#optional-parameters-).
-ANT-style wildcards are supported.
-
 #### Configuring the Path to the dotTEST Installation Directory
-If dottestcli.exe is not on PATH, you can configure the path to the installation directory of Parasoft dotTEST, which contains dottestcli.exe, by adding the `installDir` parameter do the pre-defined action.
+If `dottestcli` executable is not on `PATH`, you can configure the path to the installation directory of Parasoft dotTEST by configuring the `installDir` parameter:
 
 ```yaml
-- name: Run dotTEST analyzer
-  uses: parasoft/run-dottest-analyzer@1.0.0
+- name: Run dotTEST
+  uses: parasoft/run-dottest-action@1.0.0
```

Comment:
Change version to 1.1.0

Reply:
I've updated the version and removed all occurrences of "analyzer".